### PR TITLE
Inteface

### PR DESCRIPTION
**Pull request by kimitaWanjohi**
In this PR I did the following:
 - Create an `app.rb` for the crud functionality.
 - Add `main.rb` file for interacting with the app.rb crud functionality.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    age = gets.chomp
+    puts 'Has parent permission? [y/n]:'
+    permission = gets.chomp.downcase
+    parent_permission = permission == 'y'
+    puts 'Classroom:'
+    classroom = gets.chomp
+    @people.push Student.new(age: age, name: name, parent_permission: parent_permission, classroom: classroom)
+    puts 'Person created successfully'
+  end
+
```
Comment (danmainah) [inline app.rb:77]: - [ ] @kimitaWanjohi I am unable to create a person(teacher or student) when I run your app, am getting the error below, kindly consider ensuring all options in the menu are working correctly.
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  def student_create
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Has parent permission? [y/n]:'
+    permission = gets.chomp.downcase
+    parent_permission = permission == 'y'
+    puts 'Classroom:'
+    classroom = gets.chomp
```
Comment (danmainah) [inline app.rb:73]: - [ ] @kimitaWanjohi I can see you added an option for a student to enter their class, however, it's not part of the requirements in the console app design shown in the sneak peek, I would highly recommend sticking to the design given which is part of the project requirement.
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @kimitaWanjohi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Added the user interface and ensured it matches the design given
✔️  Ensured the user is able to add and view books.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Whoistolu — COMMENTED** [review]
Hi @kimitaWanjohi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project, that has been merged. Kindly reverse the merge before requesting a re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Whoistolu — COMMENTED** [review]
Hi @kimitaWanjohi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project, that has been merged. Kindly reverse the merge before requesting a re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Strangeal — COMMENTED** [review]
Hi @kimitaWanjohi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, that has been merged. Kindly reverse the merge before requesting a re-review.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**